src/Pages/Delivery/Delivery.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import './Delivery.scss';
+
+class Delivery extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      UserName: '',
+      UserPhone: '',
```

Comment:
- 첫글자 대문자는 컴포넌트 네이밍을 위한 컨벤션 입니다!
```suggestion
      userName: '',
      userPhone: '',
```

Reply:
수정하였습니다.